tests/unit/test_introspection.py

```diff
@@ -14,7 +14,7 @@ async def test_introspection___schema_resolver():
     sch = await __schema_resolver(None, None, None, info)
 
     assert sch == info.schema
-    assert info.is_introspection is True
+    assert info.is_introspection
```

Comment:
This is the only change I'm not really sure about, I really wanna be sure that `info.is_introspection` is a boolean and that it's value is 'True', asserting this way only assure us that it is != None or something that eval to None.

Reply:
It is unnecessary to compare boolean values to `True` or `False` in the test of an if condition. Removing these unnecessary checks makes the code slightly shorter and easier to parse.

Reply:
Yeah, But this tests makes sure that `info.is_introspection` IS a boolean by asserting that it is TRUE.
Imagine tomorrow, I make a change and sudently, `info.is_introspection` is set to 32, if this is the case I want this test to fail. With your change, it'll pass.

Reply:
Yes if we are in this case then `is TRUE` is required here to bypass that issue, Looks great this idea 💡.
Let me Revert it and then we can see 🚀

Reply:
@abusi Thanks ⚡️
The Issue is fixed

Reply:
AA Yess i forget 😅, No problem now its fixed 🚀